auto-value-firebase/src/main/java/me/mattlogan/auto/value/firebase/AutoValueFirebaseExtension.java

```diff
@@ -90,18 +103,39 @@ public String generateClass(Context context, String classNameString, String clas
     return JavaFile.builder(packageName, generatedClass).build().toString();
   }
 
+  static List<FieldSpec> generateAdapterFields(Map<String, TypeName> types) {
+    List<FieldSpec> fieldSpecs = new ArrayList<>();
+
+    for (String key : types.keySet()) {
+      TypeName typeName = types.get(key);
+      if(typeHasAdapter(typeName)){
```

Comment:
Can you make this look like `if (typeHasAdapter(typeName)) {` to match the existing code style? There are 10 or so of these sprinkled around. Thanks!